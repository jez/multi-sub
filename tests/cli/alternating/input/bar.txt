this line should not be changed
this line should be changed
