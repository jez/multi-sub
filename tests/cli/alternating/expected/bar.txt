this line should not be changed
THIS LINE should be changed
